src/funcTest/groovy/pl/droidsonroids/gradle/pitest/functional/AcceptanceTestsInSeparateSubprojectFunctionalSpec.groovy

```diff
@@ -2,18 +2,62 @@ package pl.droidsonroids.gradle.pitest.functional
 
 import groovy.transform.CompileDynamic
 import nebula.test.functional.ExecutionResult
+import spock.util.environment.RestoreSystemProperties
+
+import java.nio.file.Paths
 
 @CompileDynamic
 class AcceptanceTestsInSeparateSubprojectFunctionalSpec extends AbstractPitestFunctionalSpec {
 
+    private String htmlReport = null
+
     void "should mutate production code in another subproject"() {
         given:
-            copyResources("testProjects/multiproject", "")
+        copyResources("testProjects/multiproject", "")
         when:
-            ExecutionResult result = runTasksSuccessfully(':itest:pitestRelease')
+        ExecutionResult result = runTasksSuccessfully(':itest:pitestRelease')
         then:
-            result.wasExecuted(':itest:pitestRelease')
-            result.standardOutput.contains('Generated 2 mutations Killed 2 (100%)')
+        result.wasExecuted(':itest:pitestRelease')
```

Comment:
Maybe it would be just easier to change settings in your IDE? ;)

https://solidsoft.wordpress.com/2014/05/20/more-compact-automatic-spock-specification-formatting-in-idea-13/

Reply:
Thanks, will check that.

Reply:
Much better :-). I just cannot "close" my comment :-/. Maybe it was possible only in GitLab?